src/utils/responseHandler.js

```diff
@@ -0,0 +1,10 @@
+const notFoundResponse = async (res, message, data = null) =>
+  res.status(404).json({ status: 404, data: { message, data } });
+
+const successResponse = async (res, code, message, data = null) =>
+  res.status(code).json({ status: code, data: { message, data } });
```

Comment:
Stop using two data properties in the response, use another meaningful name instead because this is confusing